eng/pipelines/templates/variables/globals.yml

```diff
@@ -2,4 +2,7 @@ variables:
   DotNetCoreSDKVersion: '3.0.100-preview5-011568'
   DotNetCoreRuntimeVersion: '2.1.10'
   OfficialBuildId: $(Build.BuildNumber)
-  skipComponentGovernanceDetection: true
\ No newline at end of file
+  skipComponentGovernanceDetection: true
+  BuildConfiguration: ''
```

Comment:
I think we will want this to be set to Debug and not empty by default. Otherwise we will not have the Configuration set correctly as we cannot default it to Debug if you set the Configuration property as a global property via the command line as those cannot be overridden in the props file.